flexport/latitude
# LDS-441 Convert all code demos in select/__demo__

## Description:
Convert all non-knob dependent demos in the `select/_demo_` folder to live demos.

## Sandbox server:
https://core-111.sandbox3.eng-dev.flexport.com/

## Jira:
[LDS-441](https://flexport.atlassian.net/browse/LDS-441)

## Manual Testing Plan:
1. Visit the Latitude site.
2. Confirm the demos on the following pages have accurate documentation and work correctly.
   * BaseReactSelect (https://www-64.sandbox2.eng-dev.flexport.com/design/components/BaseReactSelect)
   * DropdownList (https://www-64.sandbox2.eng-dev.flexport.com/design/components/DropdownList)
   * MultiselectInput (https://www-64.sandbox2.eng-dev.flexport.com/design/components/MultiselectInput)
   * SearchableSelectInput (https://www-64.sandbox2.eng-dev.flexport.com/design/components/SearchableSelectInput)
   * SelectInput (https://www-64.sandbox2.eng-dev.flexport.com/design/components/SelectInput)
